c_emulator/riscv_rvfi_callbacks.c

```diff
@@ -0,0 +1,38 @@
+#include "riscv_config.h"
+
+int zrvfi_write(uint64_t addr, int64_t width, lbits value);
```

Comment:
In future these should be in C, not Sail. I think it's probably best to do in a future PR but can you add

```
// TODO: Move these implementations to C.
```